test/Unit/Util/PhpstormHelperTest.php

```diff
@@ -146,75 +192,77 @@ public function providePhpstormCases(): Generator
-        $expected[] = '--configuration';
-        $expected[] = '/home/user/repos/test/phpunit.xml';
-        $expected[] = '--teamcity';
+        $expectedArgv   = [];
+        $expectedArgv[] = '--coverage-clover';
+        $expectedArgv[] = '/home/user/repos/test/coverage.xml';
+        $expectedArgv[] = '--configuration';
+        $expectedArgv[] = '/home/user/repos/test/phpunit.xml';
+        $expectedArgv[] = '--teamcity';
+
+        $expectedBinary = $paratestBinary;
 
-        yield 'with -dxdebug.mode=coverage and no wrapper run ParaTest' => [
+        yield 'with -dxdebug.mode=coverage' => [
             $argv,
-            $expected,
-            $paratestBinary,
+            $expectedArgv,
             $paratestBinary,
+            $expectedBinary,
         ];
 
+        /**
+         * Additionally, with PCov, the --passthru-php option must be used to
+         * enable the sub-processes to report coverage - see README.md#pcov
+         */
         $argv   = [];
-        $argv[] = $paratestBinary;
+        $argv[] = $phpStormHelperBinary;
         $argv[] = '-dpcov.enabled=1';
         $argv[] = $phpunitBinary;
-        $argv[] = '--runner';
-        $argv[] = 'WrapperRunner';
+        $argv[] = '--passthru-php=\'-d\' \'pcov.enabled=1\'';
         $argv[] = '--coverage-clover';
         $argv[] = '/home/user/repos/test/coverage.xml';
         $argv[] = '--configuration';
         $argv[] = '/home/user/repos/test/phpunit.xml';
         $argv[] = '--teamcity';
 
-        $expected   = [];
-        $expected[] = $paratestBinary;
-        $expected[] = '--runner';
-        $expected[] = 'WrapperRunner';
-        $expected[] = '--coverage-clover';
-        $expected[] = '/home/user/repos/test/coverage.xml';
-        $expected[] = '--configuration';
-        $expected[] = '/home/user/repos/test/phpunit.xml';
-        $expected[] = '--teamcity';
-
-        yield 'with -dpcov.enabled=1 run ParaTest' => [
+        $expectedArgv   = [];
+        $expectedArgv[] = '--passthru-php=\'-d\' \'pcov.enabled=1\'';
+        $expectedArgv[] = '--coverage-clover';
+        $expectedArgv[] = '/home/user/repos/test/coverage.xml';
+        $expectedArgv[] = '--configuration';
+        $expectedArgv[] = '/home/user/repos/test/phpunit.xml';
+        $expectedArgv[] = '--teamcity';
+
+        $expectedBinary = $paratestBinary;
+
+        yield 'with -dpcov.enabled=1' => [
             $argv,
-            $expected,
-            $paratestBinary,
+            $expectedArgv,
             $paratestBinary,
+            $expectedBinary,
         ];
 
+        /**
+         * Sometimes, the phpunit binary is not in the expected location, so it
+         * needs to be able to locate it elsewhere
+         */
         $argv   = [];
-        $argv[] = $paratestBinary;
+        $argv[] = $phpStormHelperBinary;
         $argv[] = sprintf('%s/bin/phpunit', uniqid());
-
-        $argv[] = '--runner';
-        $argv[] = 'WrapperRunner';
-        $argv[] = '--no-coverage';
```

Comment:
Why these options have been removed from this test case?

Reply:
Most likely it's related to the PhpStorm runner. This is the output you get when you go through the debugger:

```
/application/vendor/brianium/paratest/src/Util/PhpstormHelper.php:40:
array(10) {
  [0] =>  string(34) "./vendor/bin/paratest_for_phpstorm"
  [1] =>  string(24) "/application/bin/phpunit"
  [2] =>  string(15) "--configuration"
  [3] =>  string(29) "/application/phpunit.xml.dist"
  [4] =>  string(8) "--filter"
  [5] =>  string(111) "/(App\\Tests\\Services\\MyService::testSomething)( .*)?$/"
  [6] =>  string(13) "--test-suffix"
  [7] =>  string(24) "MyService.php"
  [8] =>  string(40) "/application/tests/Services/MyService"
  [9] =>  string(10) "--teamcity"
}
PHP Fatal error:  Uncaught RuntimeException: Missing path to 'vendor/brianium/paratest/bin/paratest' in /application/vendor/brianium/paratest/src/Util/PhpstormHelper.php:67
Stack trace:
#0 /application/vendor/brianium/paratest/src/Util/PhpstormHelper.php(42): ParaTest\Util\PhpstormHelper::getArgvKeyFor()
#1 /application/vendor/brianium/paratest/bin/paratest_for_phpstorm(9): ParaTest\Util\PhpstormHelper::handleArgvFromPhpstorm()
#2 /application/vendor/bin/paratest_for_phpstorm(115): include('/application/ve...')
#3 {main}
  thrown in /application/vendor/brianium/paratest/src/Util/PhpstormHelper.php on line 67
ERROR: 255

Process finished with exit code 255
```
You can check my full comment here:  https://github.com/paratestphp/paratest/issues/695#issuecomment-1271889215

Reply:
@Slamdunk this test is for phpunit in an unexpected location. My thought was that each test should only use the options needed for that test. Here the wrapper is not one of the options being tested, so I removed it.